{"stdout":"","stderr":"npm ERR! code ERESOLVE\nnpm ERR! ERESOLVE could not resolve\nnpm ERR! \nnpm ERR! While resolving: learn-meter@0.1.0\nnpm ERR! Found: next@15.0.0-canary.178\nnpm ERR! node_modules/next\nnpm ERR!   next@\"^15.0.0-canary.178\" from the root project\nnpm ERR! \nnpm ERR! Could not resolve dependency:\nnpm ERR! @sentry/nextjs@\"^8\" from the root project\nnpm ERR! \nnpm ERR! Conflicting peer dependency: next@15.1.6\nnpm ERR! node_modules/next\nnpm ERR!   peer next@\"^13.2.0 || ^14.0 || ^15.0.0-rc.0\" from @sentry/nextjs@8.54.0\nnpm ERR!   node_modules/@sentry/nextjs\nnpm ERR!     @sentry/nextjs@\"^8\" from the root project\nnpm ERR! \nnpm ERR! Fix the upstream dependency conflict, or retry\nnpm ERR! this command with --force or --legacy-peer-deps\nnpm ERR! to accept an incorrect (and potentially broken) dependency resolution.\nnpm ERR! \nnpm ERR! \nnpm ERR! For a full report see:\nnpm ERR! /Users/amirghari/.npm/_logs/2025-02-03T14_36_08_912Z-eresolve-report.txt\n\nnpm ERR! A complete log of this run can be found in: /Users/amirghari/.npm/_logs/2025-02-03T14_36_08_912Z-debug-0.log\n"}